server/testutils/factories/oasysSections.ts

```diff
@@ -20,10 +20,10 @@ export default Factory.define<OASysSections>(() => ({
 export const roshSummaryFactory = Factory.define<OASysQuestion>(options => ({
   questionNumber: options.sequence.toString(),
   label: faker.helpers.arrayElement([
-    'Who is at risk?',
-    'What is the nature of the risk?',
-    'When is the risk likely to be greatest?',
-    'What circumstances are likely to increase risk?',
+    'Who is at risk',
+    'What is the nature of the risk',
+    'When is the risk likely to be greatest',
+    'What circumstances are likely to increase risk',
```

Comment:
Can these changes be rolled into the commit when you created the factories? I don't think they belong here